pkg/bootloader/grub.go

```diff
@@ -328,6 +328,15 @@ func (g *Grub) installKernelInitrd(rootPath, espDir, subfolder, snapshotID, kern
 		return nil, fmt.Errorf("copying kernel '%s': %w", kernel, err)
 	}
 
+	// Copy kernel .hmac in order to enable FIPS.
+	kernelHmac := filepath.Join(filepath.Dir(kernel), ".vmlinuz.hmac")
```

Comment:
this should also check for `.Image.hmac` (for aarch64).

Reply:
Updated to use the kernel filename to look for the .hmac file.